operator-pipeline-images/operatorcert/entrypoints/index.py

```diff
@@ -129,6 +139,27 @@ def publish_bundle(
         [build.get("state") == "complete" for build in response["items"]]
     ):
         raise Exception("IIB build failed")
+    else:
+        extract_manifest_digests(index_versions, output, response)
+
+
+def extract_manifest_digests(
+    index_versions: List[str],
+    output: str,
+    response: Dict[str, Any],
+):
+
+    LOGGER.info("Extracting manifest digests for signing...")
+    manifest_digests = []
+    # go through each version to ensure order is the same as the indices list
+    for version in index_versions:
+        for build in response["items"]:
+            if build["from_index"].endswith(version):
+                digest = build["from_index_resolved"].split("@")[-1]
```

Comment:
Should we use `index_image` and `index_image_resolved` from the IIB response instead? The "from index" is the index prior to the addition of the bundle.

Reply:
Oh thanks for catching that. Fixed.